src/main/utils.ts

```diff
@@ -133,3 +136,34 @@ export const openFilePicker = (): Promise<
     }
   });
 };
+
+export const cleanup = async () => {
+  const captainProcess = global.captainProcess as ChildProcess;
+  log.info(
+    "Cleanup function invoked, is captain running? ",
+    !(captainProcess?.killed ?? true),
+  );
+  if (captainProcess && captainProcess.exitCode === null) {
+    const success = killCaptain();
+    if (success) {
+      global.captainProcess = null;
+      log.info("Successfully terminated captain :)");
+    } else {
+      log.error("Something went wrong when terminating captain!");
+    }
+  }
+};
+
+export const loadFileFromFullPath = async (
+  filePath: string,
+): Promise<string> => {
+  return fs.readFileSync(filePath, { encoding: "utf-8" }).toString();
+};
+
+export const saveFileToFullPath = async (
+  filePath: string,
+  content: string,
+): Promise<boolean> => {
+  fs.writeFileSync(filePath, content);
+  return true;
+};
```

Comment:
why return boolean if it's always going to be true?

Reply:
Fixed! Switched to the `Result` type instead.